src/nupic/proto/LinkProto.capnp

```diff
@@ -10,4 +12,7 @@ struct LinkProto {
 
   destRegion @4 :Text;
   destInput @5 :Text;
+
+  outputElementType @6 :Int32; # NTA_BasicType: type of output elements in output array
```

Comment:
Can this be determined by checking which field in `delayedOutputs` instances of `ArrayProto` is set?